roadmap.md

```diff
@@ -20,9 +20,6 @@ The PELUX project aims to provide biannual releases. The first release was done
 in January 2018. For already released versions, see the [downloads](/downloads)
 page.
 
-### PELUX 1.1 [April 2018]
-Bugfix release
-
 ### PELUX 2.0 [June 2018]
 Content of this release is under discussion. Tentatively the topics in focus will be:
 - Intel ARP board support
```

Comment:
I am not sure if the NVIDIA board is supported

Reply:
it is apparently not

Reply:
Then we should remove that point as well